packages/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/ReactExtension.kt

```diff
@@ -137,6 +137,22 @@ abstract class ReactExtension @Inject constructor(project: Project) {
    */
   var enableHermesForVariant: (BaseVariant) -> Boolean = { enableHermes.get() }
 
+  /**
+   * Functional interface specify flags for Hermes on specific [BaseVariant] Default: will
+   * return [hermesFlagsRelease] for Release variants and [hermesFlagsDebug] for Debug variants.
+   */
+  var hermesFlagsForVariant: (BaseVariant) -> List<String> = {
+    variant -> if (variant.name.toLowerCase().contains("release")) hermesFlagsRelease.get() else hermesFlagsDebug.get()
+  }
+
+  /**
+   * Functional interface to delete debug files only on specific [BaseVariant] Default: will
+   * return True for Release variants and False for Debug variants.
+   */
+  var deleteDebugFilesForVariant: (BaseVariant) -> Boolean = {
+    variant -> variant.name.toLowerCase().contains("release")
```

Comment:
```suggestion
    variant -> variant.isRelease
```

Reply:
After making that change the build is failing with `Cannot access 'isRelease': it is private in file`

Reply:
I think it makes sense to make it internal to expose this functionality :) 
@cortinico we should consider moving it to some common utils

Reply:
Yup basically what @ShikaSD mentioned. If you want to export it to a `VariantUtil.kt` file as well, that would be fine. As long as it's `internal` all is good 👌

Reply:
It's passing now after making it internal.